firmware/README.md

```diff
@@ -23,11 +23,7 @@ Then reboot or reload udev rules:
 
 ## Build firmware
 
-In u2if/firmware/, clone pico-sdk (v1.1.0):
- - git clone https://github.com/raspberrypi/pico-sdk.git
- - cd pico-sdk
- - git reset --hard 1.1.0
```

Comment:
Keep this also? Wasn't there an alternative using checkout?

Reply:
That's completely taken care of by using a submodule. You don't need to clone it separately or check it out separately.